lib/AST/ASTMangler.cpp

```diff
@@ -2870,7 +2874,7 @@ void ASTMangler::appendEntity(const ValueDecl *decl, StringRef EntityOp,
     appendOperator("Z");
 }
 
-void ASTMangler::appendEntity(const ValueDecl *decl) {
+void ASTMangler::appendEntity(const ValueDecl *decl, bool isAsyncHandlerBody) {
```

Comment:
Should this `bool` be a `FunctionManglingKind` instead to make it clearer?

Reply:
I thought of that, but at the end decided to pass a "bool", because FunctionManglingKind has 3 cases.